internal/build/pod/maker.go

```diff
@@ -134,6 +134,7 @@ func (m *maker) podSpec(mld *api.ModuleLoaderData, containerImage string, pushIm
 		RestartPolicy: v1.RestartPolicyNever,
 		Volumes:       volumes(mld.ImageRepoSecret, buildConfig),
 		NodeSelector:  selector,
+		Tolerations:   mld.Tolerations,
```

Comment:
I was thinking about it, but I think we do need it.

For example:
* If it needs to be built on a specific arch and we set the selector of the build pod to that node (which is currently tainted)
* In the extreme case in which all nodes are tainted simultaneously (there is no reason for us to block this option).

Reply:
tainting all nodes simultaneously does not make sense, it will cause a lot of other issues within the cluster.  I agree regarding the different arch scenario

Reply:
Thanks @yevgeny-shnaidman @ybettan . As discussed in this thread, as the tolerance addition is controlled by other controllers/users, the ideal scenario would be adding taints to nodes in an orderly fashion to get the upgrades done.